spark/src/main/scala/org/apache/mahout/sparkbindings/drm/CheckpointedDrmSpark.scala

```diff
@@ -165,7 +168,14 @@ class CheckpointedDrmSpark[K: ClassTag](
       else if (classOf[Writable].isAssignableFrom(ktag.runtimeClass)) (x: K) => x.asInstanceOf[Writable]
       else throw new IllegalArgumentException("Do not know how to convert class tag %s to Writable.".format(ktag))
 
-    rdd.saveAsSequenceFile(path)
```

Comment:
It seems that the solution could be to map the DrmRdd keys and values to their respective Writables before calling `.saveAsSequenceFile(path)`.    We can use the k2wFunc for the Key, and the value will always be a Vector for Drms so simply map to a VectorWritable.     This is essentially what is done internally in `.saveAsSequenceFile` for standard Writables.
```
    // convert RDD to Writables
    rddInput.toDrmRdd().map(x => (k2wFunc(x._1), v2w(x._2)) )
                       .saveAsSequenceFile(path)
```

However the above gives a runtime exception since the Writables are not Java Serializable: ```org.apache.spark.SparkException: Task not serializable```.  Any thoughts on how to do this correctly?